scripts/nuget.cake

```diff
@@ -0,0 +1,76 @@
+#load "runhelpers.cake"
+
+using System.Net;
+
+/// <summary>
+/// Downloads and unzips a NuGet package directly without any dependencies.
+/// </summary>
+void DownloadNuGetPackage(string packageID, string version, string outputDirectory, string feedUrl)
```

Comment:
I think the latest cake build handles these cases, but I'll circle back and look at that this weekend (god I hope)